lighting_import_attachment/models/lighting_import_attachment.py

```diff
@@ -0,0 +1,30 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class LightingImportAttachment(models.Model):
+    _name = "lighting.import.attachment"
+    _description = "Lighting Import Attachment"
+
+    name = fields.Char()
+    date = fields.Date()
+    state = fields.Selection(
+        selection=[
+            ("draft", "Draft"),
+            ("in_progress", "In Progress"),
+            ("done", "Done"),
+        ],
+        default="draft",
+    )
+    file_ids = fields.One2many(
+        comodel_name="lighting.import.attachment.file",
+        inverse_name="attachment_id",
+    )
```

Comment:
Define `required`, `tracking`, `cascade`, etc... and sql constraints (if any) at the very beginning of coding, if you don't there's a very high chances you forget.